Resources/Prototypes/Recipes/Reactions/drinks.yml

```diff
@@ -1230,4 +1230,4 @@
     ZombieBlood:
       amount: 1
   products:
-    ZombieCocktail: 4
\ No newline at end of file
+    ZombieCocktail: 4
```

Comment:
This added newline is so minor I really don't want to have to faff removing it...